feature-libs/product-configurator/rulebased/components/overview-menu/configurator-overview-menu.component.ts

```diff
@@ -34,6 +34,7 @@ export class ConfiguratorOverviewMenuComponent implements AfterViewInit {
   protected readonly OV_MENU_ITEM = '-ovMenuItem';
   protected readonly OV_GROUP = '-ovGroup';
   protected readonly ACTIVE_CLASS = 'active';
+  protected readonly MENU_ITEM_HEIGHT = 39.5;
```

Comment:
Please document what this is about. I suppose this has a connection to styling attributes (we should mention this as well)

Reply:
Done!